assets/styles/landing.scss

```diff
@@ -0,0 +1,33 @@
+.body {
+    border-top: 8px solid $cream-can;
+    font-family: 'Open Sans';
+    margin: 0;
+}
+
+.body .principal {
```

Comment:
one of the advantages of sass is selector nesting, use it for this

http://sass-lang.com/guide